src/space-commander/space-commander-main.cpp

```diff
@@ -25,6 +27,7 @@ const char ERROR_EXECUTING_COMMAND = '2';
 // Declarations
 static void out_of_memory_handler();
 static int perform(int bytes);
+static void validate();
```

Comment:
I agree that the commands should do validation and return status, but this specific validate function is meant to run at start-up and log warnings if the state of the commander is not as expected.
I removed the exit(1) to simply log an unstable state error... so at least we'll have some clues of what is going on